components/Footer/__tests__/__snapshots__/Footer.test.js.snap

```diff
@@ -199,7 +199,7 @@ exports[`Footer should render with no props passed 1`] = `
     <div
       className="row copyright"
     >
-      Copyright 
+      &#169; 
```

Comment:
You don't want to manually edit snapshot files. Just run `yarn test:update-snaps` and commit the changes.